.linthtmlrc.json

```diff
@@ -1,3 +1,49 @@
 {
-  "extends": "@mate-academy/linthtml-config"
```

Comment:
Please do not add such files in future PRs

Reply:
It is pretty interesting. Because I didn't add it by myself. I don't even know where it comes from)

Reply:
Probably you've just add all files by `git add .` 🙂 Better to add src files only